app/views/users/profile.html.erb

```diff
@@ -105,28 +105,57 @@
 
   <div id="social-links">
 
-    <div class="profile_icons d-flex justify-content-center" style="gap: 8px">
-      <% if @twitter.nil? == false %>
-        <a href="<%= @twitter %>" >
-          <i class="fa fa-twitter fa-3x" aria-hidden="true"></i>
-        </a>
-      <% end %>
-      <% if @facebook.nil? == false %>
-        <a href="<%= @facebook %>" >
-          <i class="fa fa-facebook fa-3x" aria-hidden="true"></i>
-        </a>
-      <% end %>
-      <% if @github.nil? == false %>
-        <a href="<%= @github %>" >
-          <i class="fa fa-github fa-3x" aria-hidden="true"></i>
-        </a>
-      <% end %>
-      <% if @instagram.nil? == false %>
-        <a href="<%= @instagram %>" >
-          <i class="fa fa-instagram fa-3x" aria-hidden="true"></i>
-        </a>
-      <% end %>
-    </div>
+        <div class="profile_icons d-flex justify-content-center" style="gap: 8px">
+          <% if @twitter_user.nil? == false %>
+          <span>
+           <a aria-label="Delete tag" data-confirm="Are you sure you want to delete it?" class="tag-delete" data-remote="true" href="/profile/tags/delete/<%= @profile_user.id %>?name=<%= @twitter_user %>" data-method="delete">
```

Comment:
@Forchapeatl The tagname needs to be prefixed with the site's name
```suggestion
           <a aria-label="Delete tag" data-confirm="Are you sure you want to delete it?" class="tag-delete" data-remote="true" href="/profile/tags/delete/<%= @profile_user.id %>?name=twitter:<%= @twitter_user %>" data-method="delete">
```

Reply:
@TildaDares Thank you, the changes have been made

Reply:
Great job @Forchapeatl!! The only left to do is remove the icon from the page without reload. You mentioned that you knew a way to do that in https://github.com/publiclab/plots2/issues/10773#issuecomment-1068866280

Reply:
@TildaDares  yes, of course